### DISCUSSION TITLE
Exercise Module 04 - Custom Datasets

### DISCUSSION THREAD

**Discussion by daniel-doradom** [mrdbourke/pytorch-deep-learning#1218] [Q&A]
When training model_2 (TinyVGG with 50 epochs), I'm not getting same results as Daniel's. The solution says it overfits, getting good accuracy for training data but not for testing data. However, in my case this doesn't happen. 

I even downloaded Daniel's code and run it in my pc and then, the results is the same as mine (w/o overfitting). Is this normal? Is everyone else not getting the overfitting or is it just me?

**Daniel's Solution:** [04_pytorch_custom_datasets_exercise_solutions.ipynb]
Epoch: 50 | train_loss: 0.0301 | **train_acc: 0.9867** | test_loss: 4.4475 | **test_acc: 0.5467**

**Mine:**
Epoch: 50 | train_loss: 1.0988 | **train_acc: 0.3467** | test_loss: 1.1026 | **test_acc: 0.3333**

**Daniel's Code on my PC:**
Epoch: 50 | train_loss: 1.0988 | **train_acc: 0.3467** | test_loss: 1.1026 | **test_acc: 0.3333**
